spec/integration/parser_process_spec.rb

```diff
@@ -0,0 +1,68 @@
+RSpec.describe 'Process parser', type: :integration do
+  let(:invalid_schemas) {
+    %w[
+      calendar
+      coming_soon
+      completed_transaction
+      external_content
+      finder
+      generic
+      hmrc_manual
+      homepage
+      local_transaction
+      mainstream_browse_page
+      policy
+      service_manual_homepage
+      special_route
+      step_by_step_nav
+      topic
+      world_location
+    ]
+  }
+
+  context 'with a list of content schemas' do
+    it 'extracts the content valid schemas' do
+      schemas = valid_schemas
+      schemas.each do |schema|
+        expect(Item::Content::Parser.extract_content(schema)).to be_a(String).or eq(nil)
+      end
+    end
+
+    it 'returns nil for unused schemas' do
+      schemas = unused_schemas
+      schemas.each do |schema|
+        expect(Item::Content::Parser.extract_content(schema)).to be_nil
+      end
+    end
+  end
+
+  def get_schema_collection
+    schemas = GovukSchemas::Schema.all(schema_type: "frontend")
+
+    schema_list = []
+    schemas.each_value do |schema|
+      schema_list << schema
+    end
+    schema_list
+  end
+
+  def unused_schemas
```

Comment:
[Minor] I would expect `unused_schemas` to be called `invalid schemas` because of the invalid schemas variable and the valid_schema method. Any reason we have gone for `unused` rather than `invalid`?